documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -170,6 +172,20 @@ You can migrate virtual machines to {virt} from the command line.
 
 include::modules/migrating-virtual-machines-cli.adoc[leveloffset=+2]
 include::modules/obtaining-vmware-fingerprint.adoc[leveloffset=+2]
+
+=== Additional authentication methods for migrations with {osp} source providers
```

Comment:
I don't think it's the right place for this, wouldn't it make more sense to place it closer to the creation of the OSP provider? maybe in the `Prerequisites` section, like we mention the manage CA that is needed for RHV in the prerequisites section of RHV providers